userspace/libpman/src/configuration.c

```diff
@@ -165,15 +169,138 @@ int pman_get_required_buffers()
 	return g_state.n_required_buffers;
 }
 
+static char byte_array[] = "BPF_TRACE_RAW_TP";
+
+bool check_location(const char* path)
+{
```

Comment:
Since this is only used in `check_location` and `byte_array` is a pretty generic name, I would move it inside the function. By making it static it will not be initialized on every call anyways, so we should be OK (also, `const` because why not?). You could also rename it to something like `bpf_trace_raw`, but I'm not as concerned with naming.
```suggestion
bool check_location(const char* path)
{
	static const char byte_array[] = "BPF_TRACE_RAW_TP";
```

Reply:
Done, thank you!